app/src/sharedTest/java/org/oppia/android/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -142,7 +142,7 @@ class InputInteractionViewTestActivityTest {
   }
 
   @Test
-  @DisableAccessibilityChecks
+  @DisableAccessibilityChecks // Disabled as this is a test file and will not be used by user
```

Comment:
Can we specify the class name rather than using `this`?
Example `// Disabled, as InputInteractionViewTestActivity is a test file and will not be used by user`

Reply:
Done